priv/templates/components/timeline.eex

```diff
@@ -312,13 +329,31 @@ defmodule <%= @module %> do
   end
   <% end %>
 
-  defp line_width("extra_small"), do: "after:w-[0.031rem]"
-  defp line_width("small"), do: "after:w-[0.063rem]"
-  defp line_width("medium"), do: "after:w-[0.094rem]"
-  defp line_width("large"), do: "after:w-[0.156rem]"
-  defp line_width("extra_large"), do: "after:w-[0.188rem]"
-  defp line_width(params) when is_binary(params), do: params
-  defp line_width(_), do: line_width("extra_small")
+  defp line_style("solid", false), do: "after:border-solid"
```

Comment:
I think we can decrese some functions

for example see this:

```elixir
defp line_style("solid", false), do: "after:border-solid"
defp line_style("solid", true), do: "border-solid"
```

#### Can be like this:
```elixir
defp line_style("solid", horizontal?), do: horizontal? && "border-solid" || "after:border-solid"
```